### PR TITLE
changes currently running

### PR DESCRIPTION
**Pull request by Hedonil**
Hi CP, I did some minor tweaks, here again for the repo.

### CODE REVIEW

File: public_html/templates/main.tpl [hunk 1]
```diff
@@ -16,7 +16,7 @@
...
-<a href="//tools.wmflabs.org/xtools/">Home</a> &middot; <a href="//tools.wmflabs.org/xtools/ec">Edit counter</a> &middot; <a href="//tools.wmflabs.org/xtools/articleinfo/">Page History Statistics</a> &middot; <a href="//tools.wmflabs.org/xtools/blame">Article blamer</a> &middot; <a href="//tools.wmflabs.org/xtools/rangecontribs">CIDR</a> &middot; <a href="//tools.wmflabs.org/xtools/ipcalc">IP calculator</a> &middot; <a href="//wiki.toolserver.org/view/User:tparis/Index">Index</a>
+<a href="//tools.wmflabs.org/xtools/">Home</a> &middot; <a href="//tools.wmflabs.org/xtools/ec">Edit counter</a> &middot; <a href="//tools.wmflabs.org/xtools/articleinfo/">Page History Statistics</a> &middot; <a href="//tools.wmflabs.org/xtools/blame">Article blamer</a> &middot; <a href="//tools.wmflabs.org/xtools/rangecontribs">CIDR</a> &middot; <a href="//tools.wmflabs.org/xtools/ipcalc">IP calculator</a> &middot; <a href="//tools.wmflabs.org/xtools/bash">IRC random quote</a> &middot; <a href="//wiki.toolserver.org/view/User:tparis/Index">Index</a>
```
Comment (cyberpower678) [inline public_html/templates/main.tpl:19]: ec redirects to pcount.  Might want to change that directly.  Also isn't toolserver going to die including the wiki?

File: public_html/templates/mainSmarty.tpl [hunk 2]
```diff
@@ -16,7 +16,7 @@
...
-<a href="//tools.wmflabs.org/xtools/">Home</a> &middot; <a href="//tools.wmflabs.org/xtools/ec">Edit counter</a> &middot; <a href="//tools.wmflabs.org/xtools/articleinfo/">Page History Statistics</a> &middot; <a href="//tools.wmflabs.org/xtools/blame">Article blamer</a> &middot; <a href="//tools.wmflabs.org/xtools/rangecontribs">CIDR</a> &middot; <a href="//tools.wmflabs.org/xtools/ipcalc">IP calculator</a> &middot; <a href="//wiki.toolserver.org/view/User:TParis/Index">Index</a>
+<a href="//tools.wmflabs.org/xtools/">Home</a> &middot; <a href="//tools.wmflabs.org/xtools/ec">Edit counter</a> &middot; <a href="//tools.wmflabs.org/xtools/articleinfo/">Page History Statistics</a> &middot; <a href="//tools.wmflabs.org/xtools/blame">Article blamer</a> &middot; <a href="//tools.wmflabs.org/xtools/rangecontribs">CIDR</a> &middot; <a href="//tools.wmflabs.org/xtools/ipcalc">IP calculator</a> &middot; <a href="//tools.wmflabs.org/xtools/bash">IRC random quote</a> &middot; <a href="//wiki.toolserver.org/view/User:TParis/Index">Index</a>
```
Comment (cyberpower678) [inline public_html/templates/mainSmarty.tpl]: Same here.

**Comment by cyberpower678** [comment]
I added some inline comments.  Have a look.

**Comment by Hedonil** [comment]
Hell, yeah, didn't notice that. I'm digging slowly into some Peachy-, WebTool- & other classes ..
not to mention the older code snippets. I'll change it with the next pull - pages.
Greets.

**Comment by cyberpower678** [comment]
Peachy is located at MW-Peachy/Peachy on GitHub.

**Pull request merged**